GPipe-GLFW/src/Graphics/GPipe/Context/GLFW/Handler.hs

```diff
@@ -234,8 +252,8 @@ instance GPipe.ContextHandler Handle where
         Call.setErrorCallback id Nothing -- id RPC because contextHandlerDelete is called only on mainthread
 
 -- Create a raw GLFW window for use by contextHandlerCreate & createContext
-createWindow :: Log.Logger -> Maybe GLFW.Window -> Maybe (GPipe.WindowBits, Resource.WindowConfig) -> IO GLFW.Window
-createWindow logger parentHuh settings = do
+createWindow :: Log.Logger -> (Int, Int) -> Maybe GLFW.Window -> Maybe (GPipe.WindowBits, Resource.WindowConfig) -> IO GLFW.Window
```

Comment:
Could you put a type alias here, and any other place where it's just a bare tuple-of-ints?

Reply:
I’ve introduced a data type with explicit names instead.